R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
Ah I see what you mean I think. So essentially we'd add 8 new entries to the styles:

estimated1EdgesEst(lty = 1, col = "blue", lwd=1, code=1, length = 0)
estimated1EdgesStat(lty = 2, col = "red1", lwd=1, code=1, length = 0)
estimated2EdgesEst(lty=1, col = "maroon", lwd=1, code=1, length = 0)
estimated2EdgesStat(lty=3, col = "red2", lwd=1, code=1, length = 0)
estimated3EdgesEst...
estimated3EdgesStat...
estimated4EdgesEst...
estimated4EdgesStat...

Then in the config function have it be something like
`estimated1Edges = list(
      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']],
                           ), ifelse(plotItem[['newSet']] == "est", styles$estimated1EdgesEst, styles$estimated1EdgesStat))
    ),`

?